src/Mod/Path/PathScripts/PathArray.py

```diff
@@ -67,6 +75,8 @@ def __setstate__(self, state):
         return None
 
     def setEditorProperties(self, obj):
+        obj.setEditorMode('JitterPercent', 0)
+        obj.setEditorMode('JitterMagnitude', 0)
```

Comment:
Interesting. What's the use-case for a jitter?

Reply:
It's an easy way to create a hand-crafted appearance.  When you have a lot of the same cut repeated over and over with small variations it looks more natural and less mechanical.